packages/ra-ui-materialui/src/input/DateInput.stories.tsx

```diff
@@ -19,7 +19,7 @@ export const Basic = () => (
 
 export const FullWidth = () => (
     <Wrapper>
-        <DateInput source="published" fullWidth />
+        <DateInput source="published" fullWidth={false} />
     </Wrapper>
 );
```

Comment:
**suggestion**: the name of this story is misleading
```suggestion
export const NonFullWidth = () => (
    <Wrapper>
        <DateInput source="published" fullWidth={false} />
    </Wrapper>
);
```

Reply:
Applied